server/src/main/java/org/elasticsearch/index/mapper/DocumentMapper.java

```diff
@@ -120,6 +122,8 @@ public DocumentMapper build(MapperService mapperService) {
     private final MetadataFieldMapper[] deleteTombstoneMetadataFieldMappers;
     private final MetadataFieldMapper[] noopTombstoneMetadataFieldMappers;
 
+    private final static Pattern NAME_WITH_WHITESPACE_CHARACTERS = Pattern.compile(".*(?:[\\t\\r\\n]+.*)+");
```

Comment:
I am not a regex-wizard. I wonder if there is a way to do this with a non-capture group ?